### DISCUSSION TITLE
Can you change the icons in the toolbar of @mantine/rte

### DISCUSSION THREAD

**Discussion by AabhasDhaubanja** [mantinedev/mantine#2712] [Q&A] [answered]
I need to change the default icons used in the toolbar. I am sorry if this is a dumb question.

![Screenshot from 2022-10-16 22-37-28](https://user-images.githubusercontent.com/43627146/196048024-07cdcca6-9cc0-452f-9ceb-0692064437ef.png)

**Comment by rtivital** [comment, marked as answer]
Currently, that is not supported. It is planned to introduce this feature in 6.0 alongside rte package revamp.
